### PR TITLE
Document supported functionality in the README

### PR DESCRIPTION
**Pull request by libutcher**
This PR duplicates the work done in Go here: https://github.com/1Password/onepassword-sdk-go/pull/118 with linked issues updated for Python.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,26 +14,62 @@
...
-1Password SDKs support authentication with [1Password Service Accounts](https://developer.1password.com/docs/service-accounts/get-started/).
-
-## Requirements
-
-The 1Password Python SDK requires:
-
-- `libssl` 3
-- `glibc` 2.32 or later
-
-If you're running a Linux distribution that still uses `libssl` version 1.1.1, such as Debian 11 or Ubuntu 20.04, you'll need to update to a later version of Linux or install the required dependencies.
```
Comment (SimonBarendse) [inline README.md:28]: Are these requirements no longer relevant?

Comment (libutcher) [reply README.md:28]: Ah shoot, my mistake! Adding back in.

Comment (libutcher) [reply README.md]: Restored. Also updated the supported features to include listing, totp, & tags for today's release.

Comment (libutcher) [reply README.md]: Just saw that you merged the JS MR today -- let me know if you'd like me to remove the v0 updates from this MR so we can merge this now and add those updates separately!

**Review by hculea — APPROVED** [review]

**Review by AndyTitu — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -85,6 +73,64 @@ if __name__ == '__main__':
...
+- [x] Concealed fields
+- [x] Text fields
+- [x] Notes
+- [x] SSH private keys (partially supported: supported in resolving secret references, not yet supported in item create/get/update)
+- [ ] SSH public keys, fingerprint and key type
+- [x] One-time passwords 
+- [x] URLs
+- [x] Phone numbers
+- [x] Credit card types
+- [ ] Credit card numbers
```
Comment (libutcher) [inline README.md:104]:
```suggestion
```

Remove credit card numbers

**Pull request merged**